README.md

```diff
@@ -21,7 +21,7 @@ to be sent to the next task.
 ## Installation
 
 ```
-$ pip install git+https://github.com/cumulus-nasa/cumulus-message-adapter-python.git
+$ pip install cumulus-message-adapter-python
```

Comment:
I can't see a good reason to *not* use the pypi deployed library, particularly as it brings in it's associated CMA.